runtime-sdk/src/modules/consensus_accounts/mod.rs

```diff
@@ -317,6 +343,7 @@ impl<Accounts: modules::accounts::API, Consensus: modules::consensus::API>
         });
     }
 
+    #[handler(message_result = "consensus.WithdrawIntoRuntime")]
```

Comment:
This one should also be updated to use the constant (`CONSENSUS_WITHDRAW_HANDLER`)

Reply:
:see_no_evil: 
fixed